tests/vmi_networking_test.go

```diff
@@ -756,11 +756,11 @@ var _ = Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 					ipv4NetworkCIDR = api.DefaultVMCIDR
 				}
 				By("Checking ping to gateway")
-				pingVirtualMachine(serverVMI, gatewayIPFromCIDR(ipv4NetworkCIDR), "\\$ ")
+				testPingFromVMI(serverVMI, gatewayIPFromCIDR(ipv4NetworkCIDR), "\\$ ")
 
 				By("Checking ping to google")
-				pingVirtualMachine(serverVMI, "8.8.8.8", "\\$ ")
-				pingVirtualMachine(clientVMI, "google.com", "\\$ ")
+				testPingFromVMI(serverVMI, "8.8.8.8", "\\$ ")
+				testPingFromVMI(clientVMI, "google.com", "\\$ ")
```

Comment:
same

Reply:
Have you addressed this in a different commit? I do not see it in the first commit.

Reply:
Thanks. I think this is a mistake I forgot to mention: 
we are pinging here "google.com" domain and not the IP 8.8.8.8.
so I don't think there is a point of referring the address here .
I will make it `"VMI %s was not able to ping google.com domain"` , so it will be clearer.

Reply:
Now that error has enough info, you can just add "fail to ping google..."

Reply:
Done.